pkg/client/dtclient/dummy_client.go

```diff
@@ -263,7 +273,7 @@ func (c *DummyClient) ListSettings(_ string, _ ListSettingsOptions) ([]DownloadS
 	return make([]DownloadSettingsObject, 0), nil
 }
 
-func (l *DummyClient) DeleteSettings(_ string) error {
+func (c *DummyClient) DeleteSettings(_ string) error {
```

Comment:
just brings the receiver names in line for all of the methods, don't know why this one was different